web/src/utils/formatting.test.ts

```diff
@@ -23,27 +23,42 @@ describe('formatCo2', () => {
     const expected = '0 t';
     expect(actual).toBe(expected);
   });
+  it('uses same unit as another value would non-zero', () => {
+    const actual = formatCo2(230_500, 2_350_000);
+    const expected = '0.2 t';
+    expect(actual).toBe(expected);
+  });
```

Comment:
Not sure I understand this test, is it supposed to be more different from the one below?